workflows/mag.nf

```diff
@@ -613,6 +621,17 @@ workflow MAG {
         ch_versions = ch_versions.mix(PRODIGAL.out.versions.first())
     }
 
+    /*
+    ================================================================================
+                                    Virus identification
+    ================================================================================
+    */
+
+    if (params.run_genomad){
+        GENOMAD(ch_assemblies, ch_genomad_db)
```

Comment:
One other minor point - would it make sense to give this subworkflow a sematic name (e.g. VIRUS_IDENTIFICATION) in case we want to add other tools down the line that aren't Genomad?

Reply:
Good point!

Reply:
Workflow has been renamed in PR I submitted to @PhilPalmer's fork!